skimage/util/montage.py

```diff
@@ -62,26 +66,36 @@ def montage2d(arr_in, fill='mean', rescale_intensity=False, grid_shape=None):
     >>> arr_out.shape
     (4, 4)
     >>> arr_out
-    array([[  0. ,   1. ,   4. ,   5. ],
-           [  2. ,   3. ,   6. ,   7. ],
-           [  8. ,   9. ,   5.5,   5.5],
-           [ 10. ,  11. ,   5.5,   5.5]])
+    array([[ 0,  1,  4,  5],
+           [ 2,  3,  6,  7],
+           [ 8,  9,  5,  5],
+           [10, 11,  5,  5]])
```

Comment:
Filled with `5` here, but mean is still `5.5` (see below). Doesn't seem correct.